packages/mobile/src/invite/saga.ts

```diff
@@ -397,13 +402,17 @@ export function* moveAllFundsFromAccount(
     `Temp account balance is ${accountBalanceWei.toString()}. Calculating withdrawal fee`
   )
   const tempAccountBalance = divideByWei(accountBalanceWei)
-  const sendTokenFeeInWei: BigNumber = yield call(getSendFee, account, currency, {
-    recipientAddress: toAccount,
-    amount: tempAccountBalance,
-    comment,
-  })
-  // Inflate fee by 10% to harden against minor gas changes
-  const sendTokenFee = divideByWei(sendTokenFeeInWei).times(1.1)
+
+  // Temporarily hardcoding fee estimate to be 1/2 a cent to save time on estimation. Actual
+  // fees are currently an order of magnitude smaller (0.0003)
+  const sendTokenFee = new BigNumber(0.005)
```

Comment:
It's a bit ironic cause this is what we used to do. Then @jeanregisser and I overhauled fee estimation across the whole app, replacing the hard-coding with a properly calculated fee. And now where we are again :)

Reply:
Suggest hoisting this contact to the top of the file for clarity